brainstormforce/ultimate-addons-for-gutenberg
# Fix: Marketing button - Text color and hover color overriding the block-library style.

### Description

- Trello - https://trello.com/c/kso2hCc5/286-marketing-button-text-color-and-hover-color-overriding-the-css

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the cover block and inside it add a Marketing button block.
- Add color and hover color to the text and content.
- Check the frontend it overrides the WordPress block library colors.
- Switch to marketing-btn-css-issue-on-frontend and check again.

### Checklist:
